src/components/App/Snackbar/snackbarReducer.ts

```diff
@@ -0,0 +1,42 @@
+import produce, { Draft } from 'immer'
+import {
+    HIDE_SNACKBAR,
+    SHOW_ERROR_SNACKBAR,
+    SHOW_SUCCESS_SNACKBAR,
+} from './snackbarActionTypes'
+import { SnackbarState } from './snackbarSelector'
+
+const initialState = {
+    snackbar: {
```

Comment:
it's already the snackbar, no need to have another level there

Reply:
I based this on @ntchou work on users

Reply:
hmm, does not seems to be a good idea either. 
so here the state is actually `state.snackbar.snackbar` which make it redondent.